apps/docs/components/Avatar/Avatar.types.ts

```diff
@@ -0,0 +1,5 @@
+export interface AvatarProps {
+  username: string;
+  ghId: string;
+  nameKr: string;
```

Comment:
```suggestion
  username: string;
  githubId: string;
  name: string;
```
`p4`
요기도 아래와 같은 의견입니다.!

Reply:
확인입니다!

Reply:
[fix: nameKr -> name](https://github.com/function-and-mountain/functional-coding-nutshell/pull/2/commits/7bd3d629c9c22de92f1ac4892380011030a38457)